### DISCUSSION TITLE
过期：lazy-20220222 & CM-20220310

### DISCUSSION THREAD

**Discussion by hooke007** [hooke007/mpv_PlayKit#89] [回收]
版本专帖

<hr /><em>Link <a href='https://github.com/hooke007/MPV_lazy/releases/tag/20220222'>mpv-lazy-20220222</a></em>

<hr /><em>Link <a href='https://github.com/hooke007/mpv.net_CM/releases/tag/5700'>mpvnet-CM-20220310</a></em>

**Comment by hooke007** [comment]
mpv至少需要2022.02.02之后的版本，其它主程序可沿用2021EX

此包可用的内核版本上限为 [git-fcc81cd](https://github.com/shinchiro/mpv-winbuild-cmake/releases/tag/20220323)  原因见 https://github.com/hooke007/MPV_lazy/discussions/101

**Comment by hooke007** [comment]
两个版本自带的yt-dlp解析部分网站存在问题，如有需要注意手动替换/更新该组件

刚发布的CM虽然加回了youtube-dl临时解决部分网站无法解析的问题，但依然是备选，编辑 ytdl_hook.conf 中的 `#ytdl_path=yt-dlp` 修改为 `ytdl_path=youtube-dl` 来使它成为首选

<s>发布的版本无法截图保存为webp/png。 自行替换为完整版内核即可（一不小心精简过头了</s> release页面已追加修正后的精简内核

**Reply by zhongfly** [reply]
最后一个能用的yt-dlp是2022.1.30，之后的2022.1.31就不能用了，即 [https://github.com/yt-dlp/yt-dlp/pull/1964](https://github.com/ytdl-patched/yt-dlp/commit/f8580bf02f66ff6606d667de3172aeacc40c01fc) 之后mpv+yt-dlp+bilibili就出问题了，尚不清楚是mpv问题，还是yt-dlp的问题

**Reply by zhongfly** [reply]
我做了下研究，应该是mpv下载视频时没有在请求头中添加referer的原因 https://github.com/mpv-player/mpv/issues/9973
如果有referer，正常播放：
```shell
mpv https://www.bilibili.com/video/BV1Tr4y1a7xJ --referrer=https://www.bilibili.com/
```
无referer，返回403

添加自动配置后可解决：
```ini
[bilibili-add-referrer]
 profile-desc = Add referrer for BiliBili video
 profile-cond = path:match("https://www.bilibili.com")~=nil
 profile-restore = copy
 referrer = https://www.bilibili.com/
```

**Comment by hooke007** [comment]
mpv文档的半成品汉化已上线 https://hooke007.github.io/
我的原主页迁移至 https://hooke007.github.io/index2
大量涉及的超链接待修复，如遇跳转错误则算正常现象（
